lib/notify_delivery_method.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require "notifications/client"
+
+class NotifyDeliveryMethod
+  def initialize(mail); end
+
+  def deliver!(mail)
+    client.send_email(payload(mail))
+  end
+
+private
+
+  def client
+    @client ||= Notifications::Client.new(notify_api_key)
```

Comment:
We can probably access the api_key from notify here and save a few lines of code since it's similar in length.